registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/code/PacketExternalStatusCode.java

```diff
@@ -0,0 +1,8 @@
+package io.mosip.registration.processor.status.code;
+
+public enum PacketExternalStatusCode {
+
+	PROCESSED, PROCESSING, UPLOAD_PENDING,
+	RESEND

```

Comment:
Spoke to Sasi, lets rename the following
PROCESSED to ACCEPTED
PROCESSING to RECEIVED
PROCESSED to REJECTED (In failure cases)

Reply:
fixed